.ansible-lint

```diff
@@ -4,3 +4,4 @@ skip_list:
 
 warn_list:
 - load-failure
+- command-instead-of-module
```

Comment:
it would be better to put these inline in the code, unless you are using a lot of these and it would clutter up the code - I'll defer to your judgement

Reply:
Fixed that inline, yum repolist is only used once